user-interface/src/main/java/life/qbic/datamanager/views/navigation/OpenSideNavItem.java

```diff
@@ -0,0 +1,61 @@
+package life.qbic.datamanager.views.navigation;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.shared.HasSuffix;
+import com.vaadin.flow.component.sidenav.SideNavItem;
+import com.vaadin.flow.dom.Element;
```

Comment:
NIT: There are unused imports here(Component, Tag etc.)
 I'd recommend to apply the IDE setting which prunes unused imports before pushing! :)